src/js/form.js

```diff
@@ -0,0 +1,124 @@
+import intlTelInput from 'intl-tel-input/index.js';
+import 'intl-tel-input/build/js/utils';
+import 'flatpickr';
+import rangePlugin from '../../node_modules/flatpickr/dist/plugins/rangePlugin';
+
+let countries_served = ["az", "ua", "ge"];
+
+function makeTitle(str) {
+    let strSplit = str.split(" ");
+    strSplit.forEach(function(word, index) {
+        if (strSplit[index]) {
+            strSplit[index] = word[0].toUpperCase() + word.slice(1);
+        }        
+    });
+    strSplit = strSplit.join(" ");
+    return strSplit;
+}
+
+function stringLimitationOnInput(element, regexExp) {
+    element.addEventListener("input", function(e) {
+        if (e.target.value === " ") {
+            e.target.value = "";
+        }
+        let input = makeTitle(e.target.value);
+        input = input.replace(regexExp, '');
+        element.value = input;
+        return element.value;
+    })
+}
+
+function restrictKeyboardInput(element) {
+    element.addEventListener("keydown", function(e) {
+        if (e.keyCode === 13) {
+            document.getElementById("peopleCount").focus();
+        } else if (e.keyCode === 9 || e.keyCode === 16) {
```

Comment:
combine both cases in one if

Reply:
behaviour expected has to be different in those 2 cases, enter -> focus on next field, tab or tab-shift -> move between inputs
```suggestion
        } else if (e.keyCode === 9 || e.keyCode === 16) {
```